### DISCUSSION TITLE
Favicon folder help

### DISCUSSION THREAD

**Discussion by 34-4** [vercel/community#1676] [Help] [answered]
### Summary

hello! i need help with a website i made in vs code i tried to upload on vercel but it get me a error about the favicon folder my html and css file are just fine without error, i would like to know how do i fix that issues thanks.

**Comment by amyegan** [comment]
Hi @34-4! Can you tell us what error you're seeing and share a screenshot of the folder structure of your repo in VS Code?

**Reply by 34-4** [reply]
Here some screenshots.
![Capture](https://user-images.githubusercontent.com/97484508/222603523-ab05a9e9-74d9-40fd-884b-1dbdeae109d9.PNG)
![Capture2](https://user-images.githubusercontent.com/97484508/222603555-3f5adf2a-eb2c-4932-b75b-dd9dc964cee1.PNG)

**Reply by 34-4** [reply]
everything is linked
![Capture](https://user-images.githubusercontent.com/97484508/222603738-c297904f-849d-4cf4-8f70-81e0f7e953fb.PNG)

**Comment by 34-4** [comment, marked as answer]
nvm i fixed it, Vercel was only accepting index.html only instead of Main.html.
